packages/grafana-ui/package.json

```diff
@@ -77,6 +77,7 @@
     "jquery": "3.7.0",
     "lodash": "4.17.21",
     "memoize-one": "6.0.0",
+    "micro-memoize": "^4.1.2",
```

Comment:
later PR i will remove memoize-one in favor of micro-memoize. by default, micro-memoize only caches one result, the same as memoize-one.